src/test/scala/mesosphere/marathon/core/launchqueue/impl/TaskLauncherActorTest.scala

```diff
@@ -371,41 +371,31 @@ class TaskLauncherActorTest extends AkkaUnitTest with Eventually {
       }
     }
 
-    "reschedule instance on provision timeout" in new Fixture {
-      Given("a provisioned instance")
-
-      val scheduledInstanceB = Instance.scheduled(f.app)
-      val provisionedInstance = scheduledInstanceB.provisioned(TestInstanceBuilder.defaultAgentInfo, f.app, f.provisionedTasks, clock.now())
-      Mockito.when(instanceTracker.instancesBySpecSync).thenReturn(InstanceTracker.InstancesBySpec.forInstances(f.scheduledInstance, provisionedInstance))
+    "resync instance on provision timeout" in new Fixture {
```

Comment:
the tests are just terrible, I decided not to write more of these because I have to exactly mimick what method on instance tracker is called at which point in time to be able to assert anything. I am not sure whether more test would bring us more value at this point :(